pyproject.toml

```diff
@@ -98,7 +98,9 @@ dependencies = [
 "dev" = [
   "pudb",
 ]
+"test" = ["pytest>6.0.0", "pytest-cov"]
```

Comment:
"test" seems to be duplicated twice. Suggest removing first instance and keeping the second one that includes "black"